data/falcon9.rb

```diff
@@ -74,22 +74,38 @@
     }
   },
   engines: {
-    number: 9,
-    type: 'merlin',
-    version: '1D+',
-    layout: 'octaweb',
-    engine_loss_max: 2,
-    propellant_1: 'liquid oxygen',
-    propellant_2: 'RP-1 kerosene',
-    thrust_sea_level: {
-      kN: 845,
-      lbf: 190000
-    },
-    thrust_vacuum: {
-      kN: 914,
-      lbf: 205500
+    first_stage: {
+      number: 9,
+      type: 'merlin',
+      version: '1D+',
+      layout: 'octaweb',
+      engine_loss_max: 2,
+      propellant_1: 'supercooled LOX',
+      propellant_2: 'RP-1 kerosene',
+      thrust_sea_level: {
+        kN: 845,
+        lbf: 190000
+      },
+      thrust_vacuum: {
+        kN: 914,
+        lbf: 205500
+      },
+      thrust_to_weight: 180.1
     },
-    thrust_to_weight: 180.1
+    second_stage: {
+      number: 1,
+      type: 'merlin',
+      version: '1D Vaccum',
+      layout: 'Gimballed',
+      engine_loss_max: 0,
+      propellant_1: 'supercooled LOX',
+      propellant_2: 'RP-1 kerosene',
+      thrust_vacuum: {
+        kN: 934,
+        lbf: 210,000
```

Comment:
* lbf can't have a comma since it's not a string
* min_throttle can't have the percent sign since it's not a string
* That's it, should be good after those fixes